src/main/java/teammates/common/datatransfer/StudentAttributesFactory.java

```diff
@@ -105,7 +105,10 @@ public StudentAttributes makeStudent(String enrollLine, String courseId) throws
 
         String paramSection;
 
-        if (hasSection && columns.length > sectionColumnIndex) {
+        /* Since the migration to spreadsheet interfaces, section header index will always be present.
+        Therefore, if the user chooses not to include a section entry in the student row during enrollment,
+        we treat it that the student has no section. */
```

Comment:
Ideally, a future reader of this comment should not need to know there was a data migration in the past. Code comments should explain the _current state_ irrespective of what happened in the past.